Codeinwp/templates-patterns-collection
# Sync button layout on hover

### Summary
It seems like in older Safari browsers ( maybe chrome too, Rodica said it was present there too) the paddings of ::before / ::after does not merge with the padding of their parent. Removing the padding-top and padding-bottom fixed the issue.

### Test instructions
This issue needs to be tested on BrowserStack if you have the latest version of Safari. It happens on 13.1 or lower
- Install neve, neve pro and template clouds
- Go to Dashboard -> Starter sites -> My Library and hover over the Sync button
- If the button doesn't appear, you might need a license that has the library enabled.


Closes #36.
